entries/a/anonaddy.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Anonaddy": {
+    "domain": "anonaddy.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://anonaddy.com/security/",
```

Comment:
The documentation element is meant to explain to users how they can enable 2FA on the service. This link doesn't explain that. Could you please find another, more relevant, link? If not, please remove this line.

Reply:
github doesn't to like me very much and is error out when I try to access my branch. let me see what I can do it access it. I am going to remove the documentation

Reply:
Hmm yes I also get the 500 error. Maybe wait an hour or so? If it doesn't work then, you might want to consider removing that branch and creating a new PR.

Reply:
Maybe this works :shrug: 
```suggestion
```